.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+env:
+    matrix:
+    - IMAGE=centos:centos6 COMPONENTS=udt,myproxy,ssh
+    - IMAGE=centos:centos7 COMPONENTS=udt,myproxy,ssh
+#    - IMAGE=fedora:25      COMPONENTS=udt,myproxy,ssh
+#    - IMAGE=fedora:26      COMPONENTS=udt,myproxy,ssh
+#    - IMAGE=fedora:27      COMPONENTS=udt,myproxy,ssh
```

Comment:
Could we either remove these comments or get explanatory text for why they're commented out?